### ISSUE TITLE
[QUESTION] rtspsrc streams data but cant see video preview.

### ISSUE THREAD

**Issue by idonahum1** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#1364]
Hi,
I modified the pipeline in the GstSample so it can consume video stream from hikivision IP camera.
when openning the viewer preview it seems like data is being transfered but there is no video preview.
![image](https://user-images.githubusercontent.com/87803057/148052956-687d136b-6e9e-49d3-a1f9-b4a36e272018.png)

I tried the following pipelines:
`                    "rtspsrc location=rtsp://192.168.0.104:554/Streaming/Channels/101 user-id=myid user-pw=mypass short-header=True !"
                    " rtph264depay !  video/x-h264,format=svc ,alignment=nal !"
                    " appsink sync=TRUE emit-signals=TRUE name=appsink-video",
                    &error);`
as well I tried:
`                    "rtspsrc location=rtsp://192.168.0.104:554/Streaming/Channels/101 user-id=myid user-pw=mypass short-header=True !"
                    " rtph264depay ! h264parse !  video/x-h264,format=svc ,alignment=nal !"
                    " appsink sync=TRUE emit-signals=TRUE name=appsink-video",
                    &error);`

adding a picture of my IPCam configuration:
![image](https://user-images.githubusercontent.com/87803057/148055938-393f9446-eec9-4791-84da-a80e129651ca.png)

I also tested if the stream works well by using sink that saves the stream into a file, and it worked.

Any ideas?

Im adding the log.

[log.txt](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/files/7807396/log.txt)

** Any design considerations/constraints **
I want to establish realtime p2p video stream from IP cameras to user interface.
the constraint must be:
1) It should be fast, with the lowest latency possible.
2) The connection must be secured, that is why I want to use AWS KVS service.

**Comment by idonahum1** [comment]
Manage to find the right pipeline. Im closing this issue.
`                    "rtspsrc location=rtsp://192.168.0.104:554/Streaming/Channels/101 user-id=user user-pw=pass short-header=True !"
                    " rtph264depay ! queue !"
                    " video/x-h264,stream-format=byte-stream,alignment=au,profile=high !"
                    " appsink sync=TRUE emit-signals=TRUE name=appsink-video",`

**Comment by gkamath** [comment]
I am having a similar issue despite trying the pipeline suggested by @idonahum1. Both camera stream and the webrtc client are running on the same network. I have also reviewed 
i) these issues #711 
ii) tried to connect it to local usb webcam #1342 

In both the situation I am able to see the bitrate going above 0 indicating that data is being transferred from master to viewer. I can also confirm that as my web cam becomes active when I start running the sample.

I ran the test by enabling the flag and here are the results. Not sure if this will help debug. 
[  FAILED  ] RtpFunctionalityTest.packingUnpackingVerifySameH264Frame (54 ms)
[----------] 1 test from RtpFunctionalityTest (54 ms total)

[----------] Global test environment tear-down
[==========] 2 tests from 2 test cases ran. (1492 ms total)
[  PASSED  ] 0 tests.
[  FAILED  ] 2 tests, listed below:
[  FAILED  ] PeerConnectionFunctionalityTest.exchangeMedia
[  FAILED  ] RtpFunctionalityTest.packingUnpackingVerifySameH264Frame
